guides/Getting-Started.md

```diff
@@ -331,17 +326,6 @@ This illustrates a few points about the way the PureScript compiler generates Ja
 
 These points are important since they mean that PureScript generates simple, understandable code. The code generation process, in general, is quite a shallow transformation. It takes relatively little understanding of the language to predict what JavaScript code will be generated for a particular input.
 
-### Compiling CommonJS Modules
-
-Spago can also be used to generate CommonJS modules from PureScript code. This can be useful when using NodeJS, or just when developing a larger project which uses CommonJS modules to break code into smaller components.
-
-To build CommonJS modules, use the `spago build` command:
-
-    $ spago build
-    ...
-    Build succeeded.
-
-The generated modules will be placed in the `output` directory by default. Each PureScript module will be compiled to its own CommonJS module, in its own subdirectory.
```

Comment:
I feel like this is still important to say but for ES modules now.

Reply:
Ok will add it again with ES modules then

Reply:
I have added it again and updated to es modules. However I think this paragraph needs a bigger rewrite, it doesn't make much sense anymore imho.